README.md

```diff
@@ -0,0 +1,457 @@
+# Omniconf [![CircleCI](https://img.shields.io/circleci/build/github/grammarly/omniconf/master.svg)](https://dl.circleci.com/status-badge/redirect/gh/grammarly/omniconf/tree/master) [![](https://img.shields.io/clojars/dt/com.grammarly/omniconf?color=teal)](https://clojars.org/com.grammarly/omniconf) [![](https://img.shields.io/badge/-changelog-blue.svg)](CHANGELOG.md)
```

Comment:
So the plan is to have both `md` and `org`, right?

Reply:
My bad, forgot to remove the org file.